modules/ROOT/pages/hdfs/hdfs-connector-reference.adoc

```diff
@@ -1,649 +1,603 @@
 
 
-
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String |  Kerberos principal. It is passed to HDFS client as the "hadoop.job.ugi" configuration entry. It can be overriden by values in configurationResources and configurationEntries. We call it username for backward compatibility reasons in terms of what will be seen in XML |  |
-| Keytab Path a| String |  Path to the <a href="https://web.mit.edu/kerberos/krb5-1.12/doc/basic/keytab_def.html">keytab file</a> associated with username. It is used in order to obtain TGT from "Authorization server". If not provided it will look for a TGT associated to username within your local kerberos cache. |  |
-| Name Node Uri a| String |  The name of the file system to connect to. It is passed to HDFS client as the {FileSystem#FS_DEFAULT_NAME_KEY} configuration entry. It can be overriden by values in configurationResources and configurationEntries. |  | x
-| Configuration Resources a| Array of String |  A java.util.List of configuration resource files to be loaded by the HDFS client. Here you can provide additional configuration files. (for example, core-site.xml) |  |
-| Configuration Entries a| Object |  A java.util.Map of configuration entries to be used by the HDFS client. Here you can provide additional configuration entries as key/value pairs. |  |
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
+| Username a| String |  Kerberos principal. It is passed to HDFS client as the "hadoop.job.ugi" configuration entry. It can be overriden by values in configurationResources and configurationEntries. We call it username for backward compatibility reasons in terms of what will be seen in XML. |  | 
+| Keytab Path a| String |  Path to the https://web.mit.edu/kerberos/krb5-1.12/doc/basic/keytab_def.html[keytab file] associated with username. It is used to obtain a ticket granting ticket (TGT) from the authorization server. If not provided it will look for a TGT associated with the username within your local Kerberos cache. |  | 
+| Name Node Uri a| String |  The name of the file system to connect to. It is passed to HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overriden by values in *Configuration Resources* and *Configuration Entries*. |  | x
+| Configuration Resources a| Array of String |  A java.util.List of configuration resource files to be loaded by the HDFS client. Here you can provide additional configuration files, for example, core-site.xml. |  | 
+| Configuration Entries a| Object |  A java.util.Map of configuration entries to use by the HDFS client. Here you can provide additional configuration entries as key and value pairs. |  | 
```

Comment:
"key-value pairs" is more standard than "key and value pairs."

Suggestions: 

You can provide additional configuration resource files in this map, for example, core-site.xml. 

A java.util.Map of configuration entries that the HDFS client can use. You can provide additional configuration entries as key/value pairs.